config/settings.local.yml.template

```diff
@@ -1,5 +1,8 @@
 # This will override what is in settings.yml, which is committed to version control.
 
+devise:
+  secret_key:
```

Comment:
In a comment here, or maybe better in `README.md`, we should include a note about setting `secret_key`or the app won't start.